features/tags_and_wrangling/brand_new_fandoms.feature

```diff
@@ -0,0 +1,43 @@
+@user @tag_wrangling
+Feature: Brand new fandoms
+
+  Background:
+    # The external works form will error if we don't have basic tags, and since
+    # we're trying to create a brand new tag, we just want to double-check that
+    # it doesn't exist.
+    Given basic tags
+      And the tag "My Brand New Fandom" does not exist
+
+  Scenario: Brand new fandoms should be visible on the uncategorized fandoms page.
+    Given I am logged in as a random user
+      And I post a work "My New Work" with fandom "My Brand New Fandom"
+      And The periodic tag count task is run
+    When I follow "Uncategorized Fandoms" within "#header"
+    Then I should see "My Brand New Fandom"
+
```

Comment:
Could we have a follow on test that checks that the count is zero if the last external work to use a tag is destroyed ?